joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/Main.scala

```diff
@@ -61,6 +61,12 @@ object Main extends App {
           println("The input path provided is not a directory `" + sourceDir + "`. Exiting.")
           System.exit(1)
         }
+
+        val filesWithKtExtension = File(sourceDir).listRecursively.filter(f => f.extension == Some(".kt"))
```

Comment:
You could re-use `io.joern.x2cpg.SourceFiles.determine` for this job like we do in other frontends already.

Reply:
Ah, yes, thanks, updated